<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compartilhamento</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Nome :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_le">
       <property name="minimumSize">
        <size>
         <width>179</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Senha :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwd_le">
       <property name="minimumSize">
        <size>
         <width>179</width>
         <height>32</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Dom√≠nio :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="domain_le">
       <property name="minimumSize">
        <size>
         <width>179</width>
         <height>32</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>1CGEO</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="cancel_bt">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_bt">
       <property name="text">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ok_bt</tabstop>
  <tabstop>cancel_bt</tabstop>
  <tabstop>name_le</tabstop>
  <tabstop>passwd_le</tabstop>
  <tabstop>domain_le</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
